pilot/prompts/development/identify_files_to_change.prompt

```diff
@@ -0,0 +1,22 @@
+You're a senior software developer implementing changes in one file in the project.
+Based on the provided instructions and full file list, identify the file
+that needs to be modified.
+
+All files in the project:
+{% for file in files %}
+
+### {{ file['path'] }}/{{ file['name'] }}
+
+```
+{{ file['content'] }}
+```
+{% endfor %}
+
+Instructions:
+{{ code_changes_description }}
+
+Output *ONLY* the file path, relative to project root, in a single Markdown code block,
```

Comment:
it would be much better to do this in function calling rather then directly in prompt. Two reasons:

1. when using function calling we check if output is correct json and if not we repeat request which means we should always get correct response format
2. we already have issues with similar prompts where different LLMs answer in different ways, so for consistency it would be better if we always use function calling

Reply:
Switched to JSON in 8e301fa1cee720bd789e977e630c8adf55676bb1